SH4RE/SH4RE/ContentView.swift

```diff
@@ -7,10 +7,19 @@
 
 import SwiftUI
 
-var screenSize: CGRect = UIScreen.main.bounds
+let screenSize: CGRect = UIScreen.main.bounds
+let customColours = [
```

Comment:
I think the only thing you need to do to call these is `Color("DarkGrey")` I don't think you should need to make a custom colours array